Tasks/Ananich Artem/Task_7/dz_7/brain.py

```diff
@@ -0,0 +1,71 @@
+import heart
+
+
+def get_all():
+    items = heart.get_all_items()
+    comma = ','
+    return '\n'.join([f'{i[0]} - {comma.join(i[1])}' for i in items])
+
+
+def add_to_basket(n):
+    basket = heart.add_to_basket()
+    basket.append(heart.katalog[n][0])
+    return basket
```

Comment:
technically it's correct (which is the best type of correct), but according to the tri-layer architecture such logic should be placed in the data layer